src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/LanguageServerConstants.cs

```diff
@@ -23,6 +23,6 @@ public static class LanguageServerConstants
 
         public const string RazorLanguageQueryEndpoint = "razor/languageQuery";
 
-        public const string RazorMapToDocumentRangeEndpoint = "razor/mapToDocumentRange";
+        public const string RazorMapToDocumentRangesEndpoint = "razor/mapToDocumentRanges";
```

Comment:
Do we have an idea of what an "average" number of ranges for this is? Say on a couple of the more complicated template files? I'm just curious how much savings we're getting for this, especially since it seems VSCode will have to react too.

Reply:
The best example of savings can be seen in the `DocumentHighlightHandler`. Highlight is very common and a reasonably sized document can have the same symbol used in multiple places. Before this change, clicking on any one of them would create a lot of mapping requests. Even if we consider all other requests as bonus, in this case, I'd say we reduce it from 5 to 1 on average.